tests/openscad_runner.py

```diff
@@ -121,20 +127,30 @@ class OpenScadRunner:
         set_variable_argument('$fa', 8) + set_variable_argument('$fs', 0.25)
 
     def __init__(self, file_path: Path):
-        self.openscad_binary_path = self.WINDOWS_DEFAULT_PATH
+        self.openscad_binary_path = self.find_openscad_binary()
         self.scad_file_path = file_path
         self.image_folder_base = Path('.')
         self.camera_arguments = None
         self.parameters = None
 
+    @classmethod
```

Comment:
I do not see the need for this to be a classmethod. lets make it static.

Reply:
If I'm reading the documentation correctly, excluding the internal changes to function signature, the largest difference is static methods would have to access variables by doing `OpenScadRunner.WINDOWS_DEFAULT_PATH`.

Meaning, class methods are equivalent to C++ static methods, and static methods are equivalent to a namespaced free function.

Reply:
I thought one of the differences is that class methods can modify the internal class state. A classmethod can return a class instance and thus is often used for factory functions. Because this function is doing nothing with the internal state of the cls object and just reading the "static" default path variables (they are not fields and are not tied to the lifetime of an instance) I think the function should be static. 

That said, I am ok with it either way as this is not the main code of this repo.